cluster/sync.sh

```diff
@@ -60,9 +59,22 @@ function deploy() {
         registry=localhost:$(./cluster/cli.sh ports registry | tr -d '\r')
     fi
 
+    manifests_dir=build/_output/manifests
```

Comment:
Since this is defined on multiple places, please define it on Makefile and pass it as env variable to these scripts.